### DISCUSSION TITLE
Best practise for headings and paragraphs

### DISCUSSION THREAD

**Discussion by duveborg** [system-ui/theme-ui#1810] [Q&A]
Is it best practise to, for example, add `variant: text.h1 `on every h1, or is it better to somehow include these styles globally? if global, how to do that?

**Comment by hasparus** [comment]
I'm not an expert on accessibility, but I suppose that for non-trivial cases heading size and styling might not always go together with heading level, so you might want to decouple them.

I wouldn't say there's a "best practice" here (gosh, I hate this term). Do whatever works for you. If don't have the need to decouple heading levels from styling, you might even go with [`:is(h1)`](https://caniuse.com/?search=is()) pseudoclass in your styles :)
